crypto/hash_test.go

```diff
@@ -1,22 +1,26 @@
 package crypto
 
 import (
+	"bytes"
 	"crypto/rand"
 	"encoding/json"
 	"sort"
+	"strings"
 	"testing"
 )
 
-// TestObject is a struct that's used for testing HashAll and HashObject. The
-// fields have to be exported so the encoder can read them.
-type TestObject struct {
-	A int
-	B byte
-	C bool
-	D string
-}
+type (
+	// TestObject is a struct that's used for testing HashAll and HashObject. The
+	// fields have to be exported so the encoder can read them.
+	TestObject struct {
+		A int
+		B byte
+		C bool
+		D string
+	}
+)
```

Comment:
no need to wrap a single struct in a `type ()` identifier, but... at least it's consistent.